mkwutil/progress/graphic.py

```diff
@@ -20,7 +20,7 @@
 
 
 jinja_env = jinja2.Environment(
-    loader=jinja2.PackageLoader("mkwutil", "graphic"),
+    loader=jinja2.PackageLoader("mkwutil", "lib", "graphic"),
```

Comment:
```suggestion
    loader=jinja2.PackageLoader("mkwutil", "lib/graphic"),
```
First argument is module name, second argument is path within module.

Reply:
Gotcha. Also need to make "lib" -> "progress"

Reply:
Done